core/src/main/scala/org/apache/spark/deploy/Client.scala

```diff
@@ -221,7 +221,6 @@ object Client {
     val conf = new SparkConf()
     val driverArgs = new ClientArguments(args)
 
-    conf.set("spark.rpc.askTimeout", "10")
```

Comment:
I don't know. @pwendell added it in https://github.com/apache/spark/commit/3d939e5fe8930368b46eb49659a9cca8f41c8768 and changed the value to 10 in another commit.